capstone/src/main/java/com/google/univiz/api/MapData.java

```diff
@@ -1 +1,13 @@
 package com.google.univiz.api;
+
+/*
+ * The MapData interface is responsible for all data management/processing specific for the maps
+ * feature of the UniViz webapp.
+ */
+public interface MapData {
+  /*
+   * Takes a list of college names and returns a CollegeMapsData object to the caller. This object
+   * contains all data relevant to the Maps feature.
+   */
+  CollegeMapsData getMapData(List<String> collegeNames);
```

Comment:
Should the return type be a `Collection<CollegeMapsData>` or `List<CollegeMapsData>`? Returning a single object does not make sense to me. Unless you think this `CollegeMapsData` data structure itself will have a list for coordinates. In which case my question is; do you think it will contain anything but a list of coordinates/name pairs?

Reply:
yes indeed-I meant List<CollegeMapsData> and I accidentally changed it to be a single object. I will change it back.